src/hotspot/share/gc/g1/g1FullCollector.hpp

```diff
@@ -58,19 +58,11 @@ class G1FullGCSubjectToDiscoveryClosure: public BoolObjectClosure {
   }
 };
 
-// Full GC Mark that holds GC id and CPU time trace. Needs to be separate
-// from the G1FullCollector and G1FullGCScope to allow the Full GC logging
-// to have the same structure as the Young GC logging.
-class G1FullGCMark : StackObj {
-  GCIdMark       _gc_id;
-  GCTraceCPUTime _cpu_time;
-public:
-  G1FullGCMark() : _gc_id(), _cpu_time() { }
-};
-
```

Comment:
I don't recall the exact reason for this, but I have a faint memory on having some discussion with its author. It's from https://bugs.openjdk.org/browse/JDK-8272651

Before this PR:
```
[32.328s][info ][gc          ] GC(47) Pause Young (Normal) (G1 Evacuation Pause) 1162M->1032M(3072M) 323.751ms
[32.328s][info ][gc,cpu      ] GC(47) User=3.22s Sys=0.00s Real=0.33s
...
[35.221s][info ][gc             ] GC(48) Pause Full (System.gc()) 1134M->660M(3072M) 2650.824ms
[35.221s][info ][gc,cpu         ] GC(48) User=23.97s Sys=1.49s Real=2.65s
```
After this PR:

```
[35.221s][info ][gc             ] GC(48) Pause Full (System.gc()) 1134M->660M(3072M) 2650.824ms
[35.221s][info ][gc,cpu         ] GC(48) User=23.97s Sys=1.49s Real=2.65s
...
[41.337s][info ][gc,cpu         ] GC(49) User=27.86s Sys=1.58s Real=3.28s
[41.337s][info ][gc             ] GC(49) Pause Full (System.gc()) 1094M->660M(3072M) 3283.658ms
```

Note that for Full-GC, the user/sys/real time is *before* GC-end, which is inconsistent with Young-GC.

Reply:
Yes, I agree with you and fixed.
I was aware of this issue(and JDK-8272651) at the beginning so mentioned at the RFR above, but...